cmd/helm/plugin_remove.go

```diff
@@ -48,8 +49,8 @@ func newPluginRemoveCmd(out io.Writer) *cobra.Command {
 }
 
 func (pcmd *pluginRemoveCmd) complete(args []string) error {
-	if err := checkArgsLength(len(args), "plugin"); err != nil {
-		return err
+	if len(args) == 0 {
+		return errors.New("please provide plugin name to remove")
```

Comment:
I think errors.New is faster fmt.Errorf, so if it is plain string, I think errors.New should be preferable, let me know if fmt.Errorf is a better option, I can replace